doc/source/overview/high_level.md

```diff
@@ -0,0 +1,508 @@
+- This algorithm is much slower for high dimensional feature spaces
+- If choosing an anchor close to a decision boundary, the method may require a considerable number of samples from $\mathcal{D}(z|A)$ and $\mathcal{D}(z|A')$ to distinguish two anchors $A$ and $A'$.
+- High dimensional feature spaces such as images need to be reduced. Typically, this is done by segmenting the image into superpixels. The choice of the algorithm used to obtain the superpixels has an effect on the anchor obtained.
+- Practitioners need to make several choices concerning parameters and domain-specific setup. For instance, the precision threshold or the method by which we sample $\mathcal{D}(z|A)$. Fortunately, alibi provides default settings for a lot of specific data types.
+
+
+#### Pertinent Positives
+
+| Model-types | Task-types     | Data-types  |
+| ----------- | -------------- | ----------- |
+| Black-box   | Classification | Tabular     |
+|             |                | Image       |
+
+
+Informally a Pertinent Positive is the subset of an instance that still obtains the same classification. These differ from anchors primarily in the fact that they aren't constructed to maximize coverage. The method to create them is also substantially different. The rough idea is to define an absence of a feature and then perturb the instance to take away as much information as possible while still retaining the original classification.
+
+Given an instance $x_0$ we set out to find a $\delta$ that minimizes the following loss:
+
+$$
+L = c\cdot L_{pred}(\delta) + \beta L_{1}(\delta) + L_{2}^{2}(\delta) + \gamma \|\delta - AE(\delta)\|^{2}_{2}
+$$
+
+where
+
+$$
+L_{pred}(\delta) = max\left\{ max_{i\neq t_{0}}[Pred(\delta)]_{i} - [Pred(\delta)]_{t_0}, \kappa \right\}
+$$
+
+$AE$ is an optional autoencoder generated from the training data. If delta strays from the original data distribution, the autoencoder loss will increase as it will no longer reconstruct $\delta$ well. Thus, we ensure that $\delta$ remains close to the original dataset distribution.
+
+Note that $\delta$ is restrained to only take away features from the instance $x_0$. There is a slightly subtle point here: removing features from an instance requires correctly defining non-informative feature values. In the case of MNIST digits, it's reasonable to assume that the black background behind each digit represents an absence of information. Similarly, in the case of color images, you might take the median pixel value to convey no information, and moving away from this value adds information.
+
+Note that we need to compute the loss gradient through the model. If we have access to the internals, we can do this directly. Otherwise, we need to use numerical differentiation at a high computational cost due to the extra model calls we need to make.
+
+**Pros**
+- Both a black and white box method
+- We obtain more interpretable results using the autoencoder loss as $\delta$ will be in the data distribution
```

Comment:
More interpetable to what? Since we're not comparing to anything else, just say "interpetable" instead of "more interpretable" ?